### ISSUE TITLE
Scale setting does not work.

### ISSUE THREAD

**Issue by AlanRibas** [SourMesen/Mesen-S#69]
This goes for both the "Mesen" emulator as well as the "Mesen-S" emulator.
Whenever I start a game on these emulators the screen size does not reach the bottom and top edge of my monitor.
That image is floating in the center of the screen.
Changing the values in the scale field can correct the problem. Making the 4: 3 image reach the edge of my monitor without stretching it.
However, this setting is not saved. Whenever I open again the emulator I need to repeat the same process. I tested the latest versions of these two emulators.

Use updated windows 10.

ps...Auto update is not working. there was a new actialization and the emulator message when I tried manual updating was that I was using the latest version. Which was not true. I took the test today.

<img width="960" alt="Sem título" src="https://user-images.githubusercontent.com/16063712/69256315-0e3c7880-0b98-11ea-948a-50423a3174f9.png">

**Comment by SourMesen** [comment]
On Mesen-S, this would typically be because you aren't using the "overscan" settings to cut the top and bottom 8 pixels of the picture (which are usually blank/black.)  Make sure you didn't enable the option to use "integer" scale values in fullscreen modes, either.

If you want to "fill" the entire monitor, you'll need to select the 16:9 aspect ratio (assuming that's what your monitor is.)  If you just want to fill it vertically, just setting the overscan correctly and ensuring the integer scale factor option is not enabled should be enough.

As is, I can't really reproduce this - entering fullscreen mode always behaves like I would expect on my end (on both emulators.)  So I would need more information on what steps you're doing, and what options you're using.

**Comment by AlanRibas** [comment]
I did what you recommended. I added the values below (photos attached) and the game image hit the top and bottom edges of my monitor. The problem is that when I close the emulator my settings get lost. The settings made by me are not maintained by the emulator (In the part of the scale option.). 
Another thing I can realize now. Once made the above settings. If I hit F11, and switch between full screen and window mode, the settings I made are also lost. Returning to the initial values of when the emulator is opened (scale).
I just wanted to make the necessary settings and no longer have to worry about that when starting a game on MESEN-S.
The setting on the OverScan tab remains saved. Even after closing and switching between window and full screen mode. The problem is in the "scale" option.

PS....I just tried the mesen (NES) emulator. And with him the same thing happens. I set things up but the information is not saved.



<img width="960" alt="A" src="https://user-images.githubusercontent.com/16063712/71547914-19e44180-2985-11ea-9216-0d5becda64b0.png">

**Comment by AlanRibas** [comment]
Another thing I can realize now. Once made the above settings. If I hit F11, and switch between full screen and window mode, the settings I made are also lost. Returning to the initial values of when the emulator is opened.
I just wanted to make the necessary settings and no longer have to worry about that when starting a game on MESEN-S.

**Comment by SourMesen** [comment]
How/where did you install Mesen/Mesen-S?  Is there any chance Mesen doesn't have the permissions required to edit the settings.xml file?  That would explain why the settings are lost.

Can you try running Mesen(-S) as an administrator and see if that helps? (If it does, this means Mesen can't update the settings.xml file)

**Comment by AlanRibas** [comment]
I installed the emulator in a folder inside drive D:
This being the way:

"D:\Jogos\EMULADORES\Nintendo (16 Bits)\Mesen-S"

I just ran the emulator with admin permissions. Nothing changed.
But I think I know what it can be.
When pressing F11, to switch from window to fullscreen mode all the setting is lost.
I think the problem is there.
Does that happen to you too?
Does using the F11 button also change the scale that was previously set?

**Comment by AlanRibas** [comment]
> How/where did you install Mesen/Mesen-S? Is there any chance Mesen doesn't have the permissions required to edit the settings.xml file? That would explain why the settings are lost.
> 
> Can you try running Mesen(-S) as an administrator and see if that helps? (If it does, this means Mesen can't update the settings.xml file)

I installed the emulator in a folder inside drive D:
This being the way:

"D:\Jogos\EMULADORES\Nintendo (16 Bits)\Mesen-S"

I just ran the emulator with admin permissions. Nothing changed.
But I think I know what it can be.
When pressing F11, to switch from window to fullscreen mode all the setting is lost.
I think the problem is there.
Does that happen to you too?
Does using the F11 button also change the scale that was previously set?



I made this video: https://www.youtube.com/watch?v=weQYNFLcCpo
